python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -317,6 +323,24 @@ def read_messages(
     def on_opened(self, opened_payload: PipesOpenedData) -> None:
         self.opened_payload = opened_payload
 
+    def update_params(
```

Comment:
can you link me to an actual usage of this? Would be very clarifying.

Reply:
I added usages with `PipesCloudWatchMessageReader` inside `PipesGlueClient` and `PipesEMRServerlessClient` in the same PR. Still to be implemented for `PipesECSClient`.